plio/views.py

```diff
@@ -346,3 +347,24 @@ def duplicate(self, request, pk):
         question.item = item
         question.save()
         return Response(self.get_serializer(question).data)
+
+
+class ImageViewSet(viewsets.ModelViewSet):
+    """
+    Image ViewSet description
+
+    list: List all image file entries
+    retrieve: Retrieve an image file entry
+    update: Update an image file entry
+    create: Create an image file entry
+    partial_update: Patch an image file entry
+    destroy: Soft delete a image file entry, along
+             with the uploaded S3 file
+    """
+
+    queryset = Image.objects.all()
+    serializer_class = ImageSerializer
+
+    def perform_destroy(self, instance):
+        instance.image_url.delete()
+        instance.delete()
```

Comment:
why is this needed if we didn't do this for VideoViewSet?

Reply:
Added `instance.image_url.delete()` to delete the image file from s3 as well. Without it, only the entry in the table is soft deleted.

Reply:
create unique name for new image file

Reply:
revert not changing from s3 and update comments on frontend about deleting from s3 too